TLM/TLM/Lifecycle/TrafficManagerMod.cs

```diff
@@ -1,30 +1,14 @@
 namespace TrafficManager.Lifecycle {
     using ColossalFramework.Globalization;
-    using CSUtil.Commons;
     using ICities;
     using JetBrains.Annotations;
     using System;
     using TrafficManager.State;
     using TrafficManager.UI;
+    using TrafficManager.Util;
 
-    public class TrafficManagerMod : IUserMod {
-#if LABS
-        public const string BRANCH = "LABS";
-#elif DEBUG
-        public const string BRANCH = "DEBUG";
-#else
-        public const string BRANCH = "STABLE";
-#endif
-
-        // Use SharedAssemblyInfo.cs to modify TM:PE version
-        // External mods (eg. CSUR Toolbox) reference the versioning for compatibility purposes
-        public static Version ModVersion => typeof(TrafficManagerMod).Assembly.GetName().Version;
-
-        // used for in-game display
-        public static string VersionString => ModVersion.ToString(3);
-
-        public static readonly string ModName = "TM:PE " + VersionString + " " + BRANCH;
-
+    public class TrafficManagerMod : ILoadingExtension, IUserMod {
+        public static string ModName => $"TM:PE {VersionUtil.VersionString} {VersionUtil.BRANCH}";
```

Comment:
What about compatibility with other mods using `ModVersion` property lie CSUR Toolbox(check comment before change)?

Reply:
doesn't csur check TMPE assembly version?

Reply:
![image](https://user-images.githubusercontent.com/26344691/117219453-f7cac100-ae0d-11eb-8f22-9d90f21aa16f.png)

Reply:
Comment was for some reason. I'll check.

Reply:
no other mod is using it either.
![image](https://user-images.githubusercontent.com/26344691/117219646-5c861b80-ae0e-11eb-984d-f0ea362074fe.png)

Reply:
// External mods (eg. CSUR Toolbox) reference the versioning for compatibility purposes
        public static Version ModVersion => typeof(TrafficManagerMod).Assembly.GetName().Version;

External mod use the version but they don't get it from the 'ModVersion' ... they get it by from assembly version.

Reply:
Yup, it's OK then